### PR TITLE
Add support for a stickers app

### PR DESCRIPTION
**Pull request by orta**
Adds @kanaabe and @owendodd's sticker app to Eigen

~~WIP because I'm waiting on Kana to change God.~~ All images are public domain.

Adds a new scheme:
screen shot 2017-01-20 at 4 06 05 pm

Will get installed (ish?) when you have Eigen installed

screen shot 2017-01-20 at 3 59 54 pm

You can do all the drag and drop stuff, and whatever provided by apple.

screen shot 2017-01-20 at 3 59 47 pm

screen shot 2017-01-20 at 3 58 55 pm

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is classed as Work in Progress</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by orta** [comment]
OK @kanaabe this is good for a quick review once green

**Comment by orta** [comment]
screen shot 2017-01-20 at 4 58 28 pm

File: CHANGELOG.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ upcoming:
       - Fixes appearance of name field in login vc - maxim
       - Fixes a bug when facebook users login without an associated email - maxim
       - Fixes a problem where users were shown both the "Contact Gallery" and "Enter Live Auction" buttons instead of just the auction one - ash
+      - Adds a sticker pack with some public domain works - kana/owen
```
Comment (kanaabe) [inline CHANGELOG.yml]: and orta!

Comment (orta) [inline CHANGELOG.yml]: I was only a conduit, I don't need the attribution

**Comment by kanaabe** [comment]
👍  thanks for helping with this! looks great

**Comment by alloy** [comment]
Awesome 🎉 

Is there any other configuration required in iTunes Connect?

**Comment by kanaabe** [comment]
We looked at iTunes Connect but it didn't look like there was any config. We saw one for iMessage Apps, but this one's a Sticker Pack.

**Comment by orta** [comment]
Yeah, this _should_ be it to my knowledge

**Comment by alloy** [comment]
Ok.

**Pull request merged**